android/player/deps.bzl

```diff
@@ -2,15 +2,17 @@ load("//jvm/dependencies:versions.bzl", "versions")
 load("@rules_player//maven:parse_coordinates.bzl", "parse_coordinates")
 
 maven = [
-    "org.jetbrains.kotlinx:kotlinx-coroutines-android:%s" % versions.kotlin.coroutines,
+    # UI helpers
+    "androidx.core:core-ktx:%s" % versions.androidx.core,
+    "androidx.appcompat:appcompat:%s" % versions.androidx.appcompat,
+    "androidx.transition:transition:%s" % versions.androidx.transition,
 
-    # TODO: Potentially externalize versions
-    "androidx.appcompat:appcompat:1.2.0",
-    "androidx.core:core-ktx:1.3.2",
-    "androidx.constraintlayout:constraintlayout:2.0.4",
-    "androidx.navigation:navigation-fragment-ktx:2.3.3",
-    "androidx.navigation:navigation-ui-ktx:2.3.3",
-    "androidx.transition:transition:1.4.1",
+    # Lifecycle
+    "androidx.lifecycle:lifecycle-runtime-ktx:%s" % versions.androidx.lifecycle,
+    "androidx.lifecycle:lifecycle-viewmodel-ktx:%s" % versions.androidx.lifecycle,
+
+    # Default fallback
```

Comment:
what's constraintlayout gotta do with default fallback? you mean the fallbackView? is that the only place we use it..?

Reply:
I would love to not have a dependency on `constraintlayout` in the base android player dep, but we are using it to build the default fallback view:

https://github.com/player-ui/player/blob/main/android/player/src/main/res/layout/default_fallback.xml#L2